### PR TITLE
Part1 review

### PR DESCRIPTION
**Pull request by l-k-**
Suggested a change to the way sensor brand and dataformat are encoded in part 1, for readability. The new approach is fully consistent with the old one.

Made some other cleanup suggestions.

I'm still working my way through part 1, but I thought I'd send a PR with what I have now, so that it doesn't get even more bloated.

Checklist before merging:

- [X] Existing tests still work (check by running the test suite, e.g. from RStudio).
- [ ] Added tests (if you added functionality) or fixed existing test (if you fixed a bug).
- [ ] Updated or expanded the documentation.
- [ ] Updated release notes in `inst/NEWS.Rd` with a user-readable summary. Please, include references to relevant issues or PR discussions.
- [ ] Added your name to the contributors lists in the `DESCRIPTION` and `CITATION.cff` files.

### CODE REVIEW

File: R/g.plot.R [hunk 1]
```diff
@@ -168,7 +168,7 @@ g.plot = function(IMP, M, I, durplot) {
     # axis(side = 1,at = ticks, labels = 1:length(ticks))
   }
   # plot data
-  if (mon == 2 | (mon == 4 & dformat == 4)) {
+  if (mon == MONITOR$GENEACTIV || (mon == MONITOR$AXIVITY && dformat == CWA)) {
```
[the remark this answers is not in the document]
Comment (l-k-) [reply R/g.plot.R]: Ugh. Thank you for catching this!

File: tests/testthat/test_convertExtEpochData.R [hunk 2]
```diff
@@ -23,36 +23,36 @@ test_that("External epoch data is correctly converted", {
...
-  
+
   # AWD
   move2folder(system.file("testfiles/Actiwatch.AWD", package = "GGIR")[1], dn)
   params_general[["windowsizes"]][1] = 60
   params_general[["dataFormat"]] = "actiwatch_awd"
   params_general[["extEpochData_timeformat"]] = "%d-%b-%Y %H:%M:%S"
-  convertEpochData(datadir = dn, studyname = "tmp_testdata", outputdir = ".",
-                              params_general = params_general)
-
```
[the remark this answers is not in the document]
Comment (l-k-) [reply tests/testthat/test_convertExtEpochData.R:37]: Got it! I undid the most recent ones, and won't cause any more of these!

File: R/GGIR.R [hunk 3]
```diff
@@ -6,38 +6,64 @@ GGIR = function(mode = 1:5, datadir = c(), outputdir = c(),
...
   # Convert paths from Windows specific slashed to generic slashes
   outputdir = gsub(pattern = "\\\\", replacement = "/", x = outputdir)
   datadir = gsub(pattern = "\\\\", replacement = "/", x = datadir)
+
   #===========================
   # Establish default start / end file index to process
   filelist = isfilelist(datadir)
-  if (dir.exists(outputdir) == FALSE) stop("\nDirectory specified by argument outputdir, does not exist")
-  derivef0f1 = FALSE
-  if (length(f0) == 0 | length(f1) == 0) {
```
[the remark this answers is not in the document]
Comment (l-k-) [reply R/GGIR.R:33]: Good point, thank you. I saw that f0 and f1 have default values, so decided that they won't be undefined, but I didn't think about the fact that the user can pass in a NULL. Addressed.

File: R/g.inspectfile.R [hunk 4]
```diff
@@ -22,83 +22,69 @@ g.inspectfile = function(datafile, desiredtz = "", params_rawdata = c(),
...
-    }
-    is.mv = ismovisens(datafile)
-    if (is.mv == TRUE) {
-      dformat = 1
+
+    extention = unlist(strsplit(filename,"[.]"))[2]
+
+    switch (extention,
+      "bin" = { dformat = FORMAT$BIN },
+      "cwa" = ,
```
Comment (vincentvanhees) [inline R/g.inspectfile.R:35]: `"cwa" = ,`

should this be

```
"cwa" = { mon = MONITOR$AXIVITY
                dformat = FORMAT$CWA
```

**Review by vincentvanhees — CHANGES_REQUESTED** [review]
One minor additional comment. Other than that all looks great.

File: R/g.inspectfile.R (hunk 4 above)
Comment (vincentvanhees) [reply R/g.inspectfile.R:35]: Update: I have now addressed this in PR #884

Comment (l-k-) [reply R/g.inspectfile.R:35]: We can add this like you suggested, but I think it's better not to. with `switch`, if we provide a name but no element to evaluate, then whenever that name matches, we fall through to the next element that isn't missing.

this is from the help for `switch`:

> If EXPR evaluates to a character string then that string is matched (exactly) to the names of the elements in .... If there is a match then that element is evaluated unless it is missing, in which case the next non-missing element is evaluated, so for example switch("cc", a = 1, cc =, cd =, d = 2) evaluates to 2.`

 So I think it's better to leave `"cwa" = ,` immediately followed by `"CWA" = { mon = MONITOR$AXIVITY<...>` This way it's clear that "cwa" and "CWA" should have an identical effect, and we don't accidentally end up with a different result for `cwa` and `CWA`.

But, either way works, so please let me know what you prefer!

Comment (vincentvanhees) [reply R/g.inspectfile.R:35]: Ok, I didn't knew that. No problem, then let's keep it this way.

**Review by vincentvanhees — APPROVED** [review]

**Pull request merged**